ui/apps/platform/src/app.tw.css

```diff
@@ -114,9 +114,22 @@ h6 {
     z-index: 9000;
 }
 
+/* Specify PatternFly light-on-dark colors explicitly for toast because variables have light-on-dark colors in Masthead. */
+
```

Comment:
Even though I am not sure it was causing an accessibility error for insufficient contrast, to me the screenshots are even more readable due to the contrast change.

Reply:
Yeah, it also surprised me that mix of classic Rox style with Toastify style passed color contrast.